nilearn/image/image.py

```diff
@@ -778,3 +778,50 @@ def math_img(formula, **imgs):
         raise
 
     return new_img_like(niimg, result, niimg.get_affine())
+
+
+def load_niimg(niimg, ensure_ndim=None, atleast_4d=False, dtype=None,
```

Comment:
I would remove from this signature "atleast_4d" and "ensure_ndim", and move one step forward "wildcards" which is more important to the end use than the control on the dtype.